publication/src/main/scala/hmda/publication/regulator/lar/RegulatorLarPublisher.scala

```diff
@@ -66,57 +68,18 @@ class RegulatorLarPublisher extends HmdaActor with LoanApplicationRegisterCassan
         S3Headers(ServerSideEncryption.AES256)
       )
 
-      val headerSource = Source.fromIterator(() =>
-        List(
-          "id|" +
-            "respondent_id|" +
-            "agency_code|" +
-            "loan_id|" +
-            "application_date|" +
-            "loan_type|" +
-            "property_type|" +
-            "purpose|" +
-            "occupancy|" +
-            "amount|" +
-            "preapprovals|" +
-            "action_taken_type|" +
-            "action_taken_date|" +
-            "msa|" +
-            "state|" +
-            "county|" +
-            "tract|" +
-            "ethnicity|" +
-            "co_ethnicity|" +
-            "race1|" +
-            "race2|" +
-            "race3|" +
-            "race4|" +
-            "race5|" +
-            "co_race1|" +
-            "co_race2|" +
-            "co_race3|" +
-            "co_race4|" +
-            "co_race5|" +
-            "sex|" +
-            "co_sex|" +
-            "income|" +
-            "purchaser_type|" +
-            "denial1|" +
-            "denial2|" +
-            "denial3|" +
-            "rate_spread|" +
-            "hoepa_status|" +
-            "lien_status\n"
-        ).toIterator).map(s => ByteString(s))
-
-      val larSource = readData(fetchSize)
+      val source = readData(fetchSize)
+        .via(filterTestBanks)
         .map(lar => lar.toCSV + "\n")
```

Comment:
When I test this by sinking the data into a local file instead of to S3, it produces a file like this:

```
2|0|1|10665|20170307|1|1|2|2|44|3|3|20170311|45460|18|167|0102.01|2|5|5|||||8|||||2|5|24|0|1|||NA|2|1

2|0|1|10654|20170307|1|1|2|1|32|3|1|20170418|45460|18|021|0403.00|2|5|5|||||8|||||2|5|25|0||||NA|2|1

2|0|1|10855|20170311|1|1|3|1|24|3|1|20170414|45460|18|167|0014.00|2|2|5|||||5|||||1|2|60|0||||NA|2|2

2|0|1|10831|20170310|1|1|3|1|8|3|1|20170410|NA|17|191|9552.00|2|2|5|||||5|||||1|2|37|0||||NA|2|1

2|0|1|10531|20170304|1|1|2|1|43|3|4|20170723|45460|18|165|0205.00|2|5|5|||||8|||||1|5|23|0||||NA|2|1
```

with an extra blank line between each csv line. Does it also do that in s3?

Reply:
Did you see this in any of the files in `S3`?. I just randomly checked 3 of the files being delivered to the production environment and their structure is fine.